### PR TITLE
[SF7] Fix the branch build

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | yes, on that branch
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -104,6 +104,9 @@
     "extra": {
         "branch-alias": {
             "dev-master": "1.10-dev"
+        },
+        "symfony": {
+            "require": "^6.4"
```
Comment (loic425) [inline composer.json:109]: it allow to fix the docker dev build

File: Dockerfile [hunk 2]
```diff
@@ -15,6 +15,8 @@ COPY . /app
 
 WORKDIR /app
 
+RUN composer global config --no-plugins allow-plugins.symfony/flex true
```
Comment (loic425) [inline Dockerfile:18]: it allow to fix the docker dev build

File: composer.json (hunk 1 above)
Comment (loic425) [reply composer.json:109]: With that configuration, it will set up dependencies with Symfony 6.4.

File: Dockerfile (hunk 2 above)
Comment (loic425) [reply Dockerfile:18]: we do not require flex in dev requirements, cause when a developer run "composer install" it will apply flex recipes.

**Review by diimpp — APPROVED** [review]

**Pull request merged**